pyproject.toml

```diff
@@ -29,6 +29,7 @@ homepage = "https://github.com/iterorganization/imas-data-dictionary"
 
 [tool.setuptools_scm]
 write_to = "data_dictionary/_version.py"
+local_scheme = "no-local-version"
```

Comment:
Very nice!
> no-local-version
    omits local version, useful e.g. because pypi does not support it
https://setuptools-scm.readthedocs.io/en/latest/extending/#available-implementations_1

[local version](https://peps.python.org/pep-0440/#local-version-identifiers)